### PR TITLE
MgidX: add optional host param

### PR DESCRIPTION
**Pull request by xmgiddev**
doc - https://github.com/prebid/prebid.github.io/pull/4932

### CODE REVIEW

File: static/bidder-info/mgidX.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-endpoint: "https://us-east-x.mgid.com/pserver"
+endpoint: "https://{{.Host}}.mgid.com/pserver"
```
Comment (gargcreation1992) [inline static/bidder-info/mgidX.yaml:1]: This change is not recommended. We highly discourage folks to use dynamic domain this way. Please review - https://github.com/prebid/prebid-server/issues/2606

**Review by gargcreation1992 — CHANGES_REQUESTED** [review]
Please review https://github.com/prebid/prebid-server/issues/2606

File: static/bidder-info/mgidX.yaml (hunk 1 above)
Comment (onkarvhanumante) [reply static/bidder-info/mgidX.yaml:1]: @xmgiddev requesting to address above comment

**Comment by onkarvhanumante** [comment]
@xmgiddev, prebid server had major release 2.0

requesting to update `mgid:master` with latest prebid master

**Comment by onkarvhanumante** [comment]
> @xmgiddev, prebid server had major release 2.0
> 
> requesting to update `mgid:master` with latest prebid master

@xmgiddev  requesting to update `mgid:master` with latest prebid master and address PR comment

**Comment by onkarvhanumante** [comment]
> > @xmgiddev, prebid server had major release 2.0
> > requesting to update `mgid:master` with latest prebid master
> 
> @xmgiddev requesting to update `mgid:master` with latest prebid master and address PR comment

@xmgiddev ping for updating this PR

**Review by bretg — CHANGES_REQUESTED** [review]
Sorry, but requiring users to enter a region is not acceptable.

I'll write up the solution in a more formal way but here's a summary:
- document for host companies that you request regional endpoints
- when they deploy your adapter's YAML file, they are responsible for using an appropriate regional endpoint

It is unrealistic to expect that multinational publishers can sort of which endpoint a given host should use. This places a heavy (or impossible) burden on the implementation of websites.

**Comment by onkarvhanumante** [comment]
closing this inactive PR. can we reopened when comments gets addressed

**Comment by xmgiddev** [comment]
Sorry for not answering for a long time
@bretg we have changed the required HOST param to the optional REGION param
@onkarvhanumante we updated with latest prebid master

**Comment by bretg** [comment]
@xmgiddev - please open a new PR from your fork.

**Pull request closed without merging**